tests/10-create.js

```diff
@@ -0,0 +1,167 @@
+    this.rowLabel = 'Test Name';
+    this.columnLabel = 'Implementation';
+    for(const [name, {endpoints, implementation}] of match) {
+      describe(name, function() {
+        const [issuer] = endpoints;
+        const verifier = implementation.verifiers.find(
+          v => v.tags.has(tag));
+        let issuedVc;
+        let proofs;
+        const verificationMethodDocuments = [];
+        before(async function() {
+          issuedVc = await createInitialVc({issuer, vc});
+          proofs = Array.isArray(issuedVc?.proof) ?
+            issuedVc.proof.filter(proof => proof.cryptosuite === cryptosuite) :
+            [issuedVc?.proof];
+          proofs.should.not.eql([], 'Expected at least one proof to have ' +
+            'cryptosuite "ecdsa-2019".');
+          const verificationMethods = proofs.map(
+            proof => proof.verificationMethod);
+          for(const verificationMethod of verificationMethods) {
+            const verificationMethodDocument = await documentLoader({
+              url: verificationMethod
+            });
+            verificationMethodDocuments.push(verificationMethodDocument);
+          }
+        });
+        it('MUST have property "cryptosuite" and be a string.', function() {
+          this.test.cell = {columnId: name, rowId: this.test.title};
+          proofs.forEach(proof => {
+            should.exist(proof.cryptosuite, 'Expected proof to have ' +
+              '"cryptosuite" property.');
+            proof.cryptosuite.should.be.a('string', 'Expected "cryptosuite" ' +
+              'property to be a string.');
+          });
+        });
+        it('The field "cryptosuite" MUST be "ecdsa-2019".', function() {
+          this.test.cell = {columnId: name, rowId: this.test.title};
+          proofs.forEach(proof => {
+            should.exist(proof.cryptosuite, 'Expected proof to have ' +
+              '"cryptosuite" property.');
+            proof.cryptosuite.should.equal(cryptosuite, 'Expected ' +
+              '"cryptosuite" property to be "ecdsa-2019".');
+          });
+        });
+        it('The "proof" MUST verify when using a conformant verifier.',
+          async function() {
+            this.test.cell = {columnId: name, rowId: this.test.title};
+            should.exist(verifier, 'Expected implementation to have a VC ' +
+              'HTTP API compatible verifier.');
+            const {result, error} = await verifier.post({json: {
+              verifiableCredential: issuedVc,
+              options: {checks: ['proof']}
+            }});
+            should.not.exist(error, 'Expected verifier to not error.');
+            should.exist(result, 'Expected verifier to return a result.');
+            result.status.should.not.equal(400, 'Expected status code to not ' +
+              'be 400.');
+            result.status.should.equal(200, 'Expected status code to be 200.');
+          });
+        it('Dereferencing "verificationMethod" MUST result in an object ' +
```

Comment:
this is good, but it might cause issues in the future if another issuer uses a `did:key` or `did:web` method we don't have a driver for. Leave the verificationMethod/ controllerDocument tests here for now and we'll refactor them when other implementers use the suite.